src/VstsSyncMigrator.Core/Execution/MigrationContext/TestPlansAndSuitesMigrationContext.cs

```diff
@@ -622,27 +622,29 @@ private ITestSuiteBase FindSuiteEntry(IStaticTestSuite staticSuite, string title
             return testSuit;
```

Comment:
The `FindSuiteEntry` method has a similar issue. (They were modified at the same time) 

Here are my proposed changes. (note the change in method signature, I'm passing in the whole source suite instead of just the name) 

```csharp

        private ITestSuiteBase FindSuiteEntry(IStaticTestSuite staticSuite, ITestSuiteBase sourceSuite)
        {
            ITestSuiteBase testSuit = (from s in staticSuite.SubSuites where s.Title == sourceSuite.Title select s).SingleOrDefault();
            if (testSuit != null)
            {
                //Check test suit is in fact the right one
                var sourceWI = Engine.Source.WorkItems.GetWorkItem(sourceSuite.Id.ToString());
                string expectedReflectedId = Engine.Source.WorkItems.CreateReflectedWorkItemId(sourceWI).ToString();
                var targetWI = Engine.Target.WorkItems.GetWorkItem(testSuit.Id.ToString());
                string workItemReflectedId = (string)targetWI.Fields[Engine.Target.Config.AsTeamProjectConfig().ReflectedWorkItemIDFieldName].Value;
                if (workItemReflectedId != expectedReflectedId)
                {
                    testSuit = null;
                }
            }
            return testSuit;
        }
```

Let me know if you'll implement them as part of this PR, or if you want to me to submit them separately.